config.json

```diff
@@ -23,6 +23,16 @@
       ]
     },
     {
+      "uuid": "e5a0bed3-90ca-4caa-a73f-2124940f84c9",
+      "slug": "isbn-verifier",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Strings"
```

Comment:
probably make a PR adding strings to all exercises that have it

Reply:
It is ok if I make that after this get merged? If that ever happens.

Reply:
Yes of course, since doing it after is one of the two possible options:

1. remove string from this PR, make string PR after this PR is merged
2. make string PR and it is merged before this PR is merged, in which case keeping string here is OK

Reply:
I was thinking on doing the first when I removed Strings.